Wordle.Domain/Guess.cs

```diff
@@ -0,0 +1,8 @@
+namespace Wordle.Domain;
+
+public class Guess
+{
+    public int Number {get; set;} = 1;
+    public string Word {get; set;} = " ";
```

Comment:
No difference, at one point I thought that perhaps this could be an empty 5 space string, and I could render out empty guesses that would eventually be replaced by the accepted Words. But I don't think I will go this route anymore so I'll update this.